build.sbt

```diff
@@ -23,12 +23,26 @@ lazy val root = (project in file(".")).enablePlugins(
 
 scalaVersion := "2.11.6"
 
+val cxfV = "2.7.5"
+val cxfUtilsV = "2.7.0"
+val exactTargetDeps = Seq(
+  "com.exacttarget" % "fuelsdk" % "1.0.3" from "https://github.com/salesforcefuel/FuelSDK-Java/releases/download/v1.0.3/fuel-java-1.0.3.jar",
+  "commons-beanutils" % "commons-beanutils" % "1.9.2",
+  "log4j" % "log4j" % "1.2.17",
+  "com.google.code.gson" % "gson" % "2.3.1",
+  "org.apache.cxf" % "cxf-bundle-minimal" % cxfV,
+  "org.apache.cxf" % "cxf-tools-common" % cxfV,
+  "org.apache.cxf" % "cxf-tools-common" % cxfV,
+  "org.apache.cxf.xjc-utils" % "xjc-utils" % cxfUtilsV,
+  "org.apache.cxf.xjc-utils" % "cxf-xjc-runtime" % cxfUtilsV
```

Comment:
This list of dependencies is pretty long! Do we actually have to declare all of it?!